src/main/java/com/crud/tasks/controller/TrelloController.java

```diff
@@ -19,9 +20,13 @@
     @RequestMapping(method = RequestMethod.GET, value = "getTrelloBoards")
     public void getTrelloBoards() {
 
-        List<TrelloBoardDto> trelloBoards = trelloClient.getTrelloBoards();
+        List<TrelloBoardDto> trelloBoards = trelloClient.getTrelloBoards().orElseThrow(() -> new NoBoardsFoundException());
 
-        trelloBoards.forEach(trelloBoardDto -> System.out.println(trelloBoardDto.getId() + " " + trelloBoardDto.getName()));
+        trelloBoards.stream()
+                .filter(e -> e.getId().length() > 0 )
```

Comment:
jak id bedzie nullem, to poleci tutaj Null Pointer

Reply:
Faktycznie, czyli z Trello może przyjść null jeśli nic nie będzie, a jeśli nie będzie to null, to samo trello pewnie dba, żeby id nie było zerowej długości?

Reply:
nie powinienes 'ufac' innym aplikacjom. Twoja aplikacja powinna byc gotowa na bledy np trello